index.html

```diff
@@ -16,46 +16,62 @@
+                    <li>Contact</li>
+                </a>
+            </ul>
+
+            <div class="nav-menu">
                 <img src="./images/menu.svg" alt="menu-icon" class="menu">
             </div>
         </nav>
 
     </header>
 
-    <section class="portfolio">
-        <h1 class="portfolio__title">I’m Ahmed Eid. Glad to see you!</h1>
-
-        <p class="portfolio__description">
-            I'm graduated from the Faculty of Computer and Informatics at Suez Canal University, where I majored
-            in Computer Science. My primary area of interest is web development, with a particular focus on the Backend
-            track.
-        </p>
-        <h2 class="contact__title">LET'S CONNECT</h2>
-        <ul class="portfolio__icons">
-            <li>
-                <a href="https://twitter.com/ahmedeid2684" rel="noopener noreferrer" target="_blank">
-                    <img src="images/twitter.svg" alt="twitter-logo">
-                </a>
-            </li>
-            <li>
-                <a href="https://www.linkedin.com/in/ahmed-eid-0018571b1/" rel="noopener noreferrer" target="_blank">
-                    <img src="images/linkedin.svg" alt="linkedin-logo">
-                </a>
-            </li>
-            <li>
-                <a href="https://github.com/ahmedeid6842" rel="noopener noreferrer" target="_blank">
-                    <img src="images/github.svg" alt="github-logo">
-                </a>
-            </li>
-        </ul>
+    <section class="portfolio" id="portfolio">
+        <div class="portfolio_paragraph">
+            <h1 class="portfolio__title">I’m Ahmed Eid.<br /> Glad to see you!</h1>
+            <p class="portfolio__description">
+                I'm graduated from the Faculty of Computer and Informatics at Suez Canal University, where I majored
+                in Computer Science. My primary area of interest is web development, with a particular focus on the
+                Backend
+                track.
+            </p>
+            <h2 class="contact__title">LET'S CONNECT</h2>
+            <ul class="portfolio__icons">
+                <li>
+                    <a href="https://twitter.com/ahmedeid2684" rel="noopener noreferrer" target="_blank">
+                        <img src="images/twitter.svg" alt="twitter-logo">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/ahmed-eid-0018571b1/" rel="noopener noreferrer"
+                        target="_blank">
+                        <img src="images/linkedin.svg" alt="linkedin-logo">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://github.com/ahmedeid6842" rel="noopener noreferrer" target="_blank">
+                        <img src="images/github.svg" alt="github-logo">
+                    </a>
+                </li>
+            </ul>
+        </div>
     </section>
-    <section class="work">
```

Comment:
- [ ] There is little spacing between the header and the works section in your desing however this is not the same as in the case in the figma template please try to design your web page to match the design of the figma template :+1: 
- figma design
![fig-d](https://user-images.githubusercontent.com/89055502/236356808-94a23c99-7339-47f0-805e-8fc545409d03.png)
- your design
![fig-s](https://user-images.githubusercontent.com/89055502/236356817-ce4be0d0-d480-473e-9d73-020813b362ef.png)